pkg/apis/pravega/v1alpha1/pravega.go

```diff
@@ -184,6 +191,27 @@ func (s *PravegaSpec) withDefaults() (changed bool) {
 	return changed
 }
 
+// PravegaImageSpec defines the fields needed for a Pravega Docker image
+type PravegaImageSpec struct {
+	ImageSpec
+}
+
+func (s *PravegaImageSpec) withDefaults(c *ClusterSpec) (changed bool) {
```

Comment:
I think `c *ClusterSpec` is actually not used in this function.

Reply:
Right. I've set the tag to `.spec.Version`.